bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -58,31 +53,30 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
       }
     }
     
-    // Case: User is logged in
-    else {
+    // Case: User is logged in but not registered (will replace else if with code checking registered)
+    else if (1!=1){
         /* TODO: Add case when user is not registered */
     }
```

Comment:
For code cleanliness, I would just remove this part here and leave the TODO as a comment after the if and else statements.

Reply:
Removed